modules/ROOT/pages/assigning-permissions.adoc

```diff
@@ -0,0 +1,430 @@
+= Assigning Permissions 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: assigning-administrator-permissions.adoc
+
+Anypoint Platform comes with predefined permissions that you can set in Anypoint Access Management to provide users with different access levels. You configure the permissions for administrators and Exchange viewers. 
+
+The Salesforce platform uses the System Administrator profile which contains permissions to perform specific tasks in Salesforce. For API Experience Hub, this profile or the AEH_Anypoint permission set is required for the user who performs the Salesforce linking.
+
+
+== Before You Begin
+Consider the permission strategy for users in the Anypoint organization and Salesforce organization.
+
+== Grant the API Experience Hub Administrator Permission
+The Organization Administration and API Experience Hub Administrator can perform all of the functions to create a portal. Only the Organization Administrator or an existing API Experience Hub Administrator can grant a user the API Experience Hub Administrator permission. Grant this permission when you want another user in your organization to perform the Salesforce linking process, create and curate the portal, manage the portal, and manage users.
+
+To grant the API Experience Hub Administrator permission to a user:
+
+. Log in to Anypoint Platform using an account with the Organization Administrator permission.
+. Click *Access Management*.
+. Click *Users*.
+. Click the username of the user that you want to add the API Experience Hub Administrator permission. 
+. Select *Permissions* and click *Add Permissions*.
+. From the *Select permissions to add for this user* search box, enter *API Experience Hub Amin* and select the checkbox.
```

Comment:
```suggestion
. In the *Select permissions to add for this user* search box, enter *API Experience Hub Admin* and select the checkbox.
```

Reply:
The intro section says that only the Org admin or existing AEH admin can grant this. I will add this to Before You Begin as well.